app/views/base/layout.scala

```diff
@@ -15,10 +15,12 @@ import lila.common.base.StringUtils.escapeHtmlRaw
 object layout:
 
   object bits:
-    val doctype                   = raw("<!DOCTYPE html>")
-    def htmlTag(using lang: Lang) = html(st.lang := lang.code, dir := isRTL.option("rtl"))
```

Comment:
The if-rtl { } functions are executed at most a few times per page load. Is it really a good idea to optimize away the attribute selector approach? Don't we want exactly one of if-ltr or if-rtl to always match regardless of classes? What about embeds?

Reply:
Yeah I'm not sure it's worth it. They do appear quite a lot in the CSS tho:

```sh
rg 'dir-(ltr|rtl)' lobby.dark.dev.css | wc -l
14
rg 'dir-(ltr|rtl)' round.dark.dev.css | wc -l
28
rg 'dir-(ltr|rtl)' analyse.study.dark.dev.css | wc -l
47
```